frontend/src/components/MeetingRoom/ReportIssueButton/ReportIssueButton.tsx

```diff
@@ -2,23 +2,30 @@ import { Tooltip } from '@mui/material';
 import { ReactElement, useRef } from 'react';
 import FeedbackIcon from '@mui/icons-material/Feedback';
 import ToolbarButton from '../ToolbarButton';
-import useIsSmallViewport from '../../../hooks/useIsSmallViewport';
 
 export type ReportIssueButtonProps = {
   handleClick: () => void;
   isOpen: boolean;
+  isOverflowButton?: boolean;
 };
 
 /**
  * ReportIssueButton Component
  *
  * Displays a clickable button to open/close the ReportIssue panel.
  * @param {ReportIssueButtonProps} props - The props for the component.
+ *  @property {Function} handleClick - click handler to open the Report Issue panel
+ *  @property {boolean} isOpen - whether the Report Issue panel is open
```

Comment:
nice, I was actually also adding these docs in the PR that I was working on 💪 😂

Reply:
Something something, great minds 🤠